ostorlab_insecure_flutter_app/lib/bug_rule.dart

```diff
@@ -5,7 +5,7 @@ abstract class BugRule {
   late BuildContext context;
 
   /// Runs the bug rule implementation asynchronously.
-  Future<void> run() async {
+  Future<void> run(String user_input) async {
```

Comment:
Please rename to input.

Reply:
[2c3d87d](https://github.com/Ostorlab/ostorlab_insecure_android_app/pull/22/commits/2c3d87d9f679ffa607a88cd4cc55780e39e06ece)